.github/actions/run-cloud-run-command-on-active-checkout/action.yml

```
@@ -26,7 +26,7 @@ runs:
       run: |
         gcloud builds submit . \
           --config=.github/cloud_builder/run_command_on_active_checkout.yaml \
-          --substitutions=_CMD="${{ inputs.cmd }}" \
+          --substitutions=_CMD="GIT_HASH=$(git rev-parse HEAD) ${{ inputs.cmd }}" \
```

Btw, substitutions replaces `_CMD`  in `GiGL/.github/cloud_builder/run_command_on_active_checkout.yaml`
We could modify that script to have another substitution off:
```bash        
        export GITHUB_REF=_ GITHUB_REF
        echo "Finished setting up environment."
        echo "========================="
        echo "Running command: ${_CMD}"
        echo "========================="
        $_CMD
        echo "Finished running command"
```


then, this command becomes: 
```suggestion
   --substitutions=_CMD="${{ inputs.cmd }}",GITHUB_REF="${{ github.ref }}" \
```

---

all variables available here: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables